server/src/controllers/contactController.ts

```diff
@@ -588,6 +608,30 @@ contactController.get('/orders/:id', async (req: Request, res: Response) => {
 contactController.put('/:id', async (req: Request, res: Response) => {
   try {
     const id = req.params.id;
+
+    await prisma.contacts.update({
+      where: {
+        contactid: Number(id),
+      },
+      data: {
+        firstname: req.body.firstname,
+        lastname: req.body.lastname,
+        email: req.body.email,
+        phone: req.body.phone,
+        address: req.body.address,
+        city: req.body.city,
+        state: req.body.state,
+        postalcode: req.body.postalcode,
+        country: req.body.country,
+        donorbadge: req.body.donorbadge,
+        seatingaccom: req.body.seatingaccom,
+        comments: req.body.comments,
+        visitsource: req.body.visitsource,
+        vip: req.body.vip,
+        volunteerlist: req.body.volunteerlist,
+        newsletter: req.body.newsletter,
+      },
+
```

Comment:
I don't believe this is necessary since it is handled by the `updateContact` call on line 640.